krabsetw.nuspec

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
could you bump the <version> tag to `4.1.13` so we can publish an updated nuget? thanks!